### ISSUE TITLE
npm install for a second time

### ISSUE THREAD

**Issue by caranicas** [reactGo/reactGo#167]
There seems to be an issue if you re run "npm install" without specified package name. I pulled the new version with nodemon and  got in running. the first thing I did was move over my dependencies and go to install them. However when i did this. it seems to have kicked off the whole rebuild process.

It isn't the biggest issue, because normally I add new packages one at a time, so it is a low priority but probably an easy fix. 

![screen shot 2016-03-11 at 1 04 55 pm](https://cloud.githubusercontent.com/assets/1971367/13710967/29ddc66a-e78a-11e5-9310-ee2c335bba72.png)

![screen shot 2016-03-11 at 1 05 13 pm](https://cloud.githubusercontent.com/assets/1971367/13710980/30d9c2fc-e78a-11e5-9dc2-7b0180f149f0.png)

**Comment by choonkending** [comment]
@caranicas 
Thanks for raising this issue.

> However when i did this. it seems to have kicked off the whole rebuild process.

The post `npm install` build process is intentional so that when we deploy it (and run `npm install` it will automatically build). The line in the config is [here](https://github.com/choonkending/react-webpack-node/blob/master/package.json#L17)

Do you think this `postinstall` is an unneeded step? I find it valuable for beginners to just install and get it built :) Let me know what you think

**Comment by caranicas** [comment]
Yeah I think it might be a bit superfluous, but more it was just unexpected. I'm not sure how inexperienced the people are that you are targeting this project at, but I would hope that if they were jumping into this scaffolding that they would have used a some build system and NPM before. They should have enough experience to know how to get a project with a build script up and running. If they don't then they have a lot of learn before they jump into something like this. 

I've been a front end dev for 5 years,  But when it wasn't Jquery spaghetti it was just some simple backbone  and coffee script. I am completely new to React and Redux. I've dabbled with node/express/passport but never anything that went to production. I've used Grunt Brunch and Gulp, but I am completely new to Webpack. I wouldn't consider myself a beginner, but when it comes to using a scaffold of this magnitude, there is definitely a lot of things that go over my head initially, But I would consider myself the target market. 

I am someone that knows what they want to build and knows what tools they want to use, but I don't quite know how to fit them all together, and don't want to take all the time (right now) to learn. So using this scaffold I can focus on working how I want and building out my App. Then I can more slowly breakdown and understand the component parts as I run into issues and can isolate my thinking.
